extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -212,20 +213,27 @@ function _checkIfCanAddMeasurementsToDisplaySet(
     newDisplaySet
   );
 
-  for (let i = 0; i < images.length; i++) {
+  for (let i = 0; i < imageIdsForDisplaySet.length; i++) {
```

Comment:
for(const imageId of imageIdsForDisplaySet) 
as that doesn't assume anything about the layout/design and just gets next until done.

Reply:
Resolved in 9d380ba8e72953e0e76d3fa1212b015973093fa2